xsnippet_api/application.py

```diff
@@ -58,7 +58,8 @@ def create_app(conf):
     # we need to import all the resources in order to evaluate @endpoint
     # decorator, so they can be collected and passed to VersionRouter
     from . import resources  # noqa
-    app = web.Application(router=router.VersionRouter(endpoint.collect()))
+    app = web.Application(router=router.VersionRouter(endpoint.collect()),
+                          middlewares=[middlewares.auth])
```

Comment:
I wonder would it be possible to specify wanted middlewares in config file? Or once we're checking user's permissions we're tied to auth? Just can't figure out right now.